controllers/stargate/stargate_controller.go

```diff
@@ -133,15 +133,17 @@ func (r *StargateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	// if a configmap is specified, we need to read its content to merge it with the generated one
-	userConfigMapContent := ""
+	userStargateCassandraYaml := ""
+	userStargateCqlYaml := ""
 	if stargate.Spec.CassandraConfigMapRef != nil {
 		userConfigMap := &corev1.ConfigMap{}
 		configMapKey := types.NamespacedName{Namespace: req.Namespace, Name: stargate.Spec.CassandraConfigMapRef.Name}
 		err := r.Get(ctx, configMapKey, userConfigMap)
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-		userConfigMapContent = userConfigMap.Data["cassandra.yaml"]
+		userStargateCassandraYaml = userConfigMap.Data["cassandra.yaml"]
+		userStargateCqlYaml = userConfigMap.Data[stargateutil.CqlConfigName]
```

Comment:
I'm trying to think of a case where we'd get a user input that should go into that Stargate cql config file.
Which settings other than `client_encryption_options` have to be passed through this file?